wordpress-mobile/WordPress-Android
# [Site Creation] Step 1: Remove verticals suggestions display

This PR removes verticals suggestions display (including custom category) from the verticals site creation step (Step 1 of 3) as part of https://github.com/wordpress-mobile/WordPress-Android/issues/11420 and https://github.com/wordpress-mobile/WordPress-Android/pull/11421. 

PR in FluxC (to remove `VerticalModel`) - https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/1509
(Temporarily pointing to commit hash for above FluxC PR which needs to be reverted before merge.) 

![device-2020-03-11-163712](https://user-images.githubusercontent.com/1405144/76410875-aca65100-63b6-11ea-921b-1eab7c938414.png)

To test:
- Go to My site
- Click Switch Site 
- Click "+" button to launch Add new site 
- Click Create WordPress.com site on Add new site 
- Choose a segment Blog/Business/Professional/Blank Canvas to open Step 1 of 3 
- Search for some query in Step 1 of 3 
- Verify no results are displayed

![verticals_suggestions](https://user-images.githubusercontent.com/1405144/76409540-82ec2a80-63b4-11ea-8b45-ef569972af76.gif)

PR submission checklist:
